src/main/java/com/project/pufferfish/screen/SpaceInvaders.java

```diff
@@ -0,0 +1,93 @@
+package com.project.pufferfish.screen;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class SpaceInvaders {
+	public int x;
+	public int y;
+	int points;
+	public boolean isAlive;
+	boolean isVisible;
+	public boolean movesRight; //Space invaders start by moving to the right
+	public static AtomicInteger Id=new AtomicInteger(); //giving the invaders a number
+	public int invaderId;
+	
+    public static void main(String[] args)
+    {
+        //System.out.print("here");
+    }
+
```

Comment:
can you please delete the commented out stuff and the invader.png, invader1.png and spaceinvaders.png